pyproject.toml

```diff
@@ -116,6 +116,7 @@ streamlit = {version = "^1.18.0", optional = true, python = ">=3.8.1,<3.9.7 || >
 psychicapi = {version = "^0.8.0", optional = true}
 cassio = {version = "^0.0.7", optional = true}
 rdflib = {version = "^6.3.2", optional = true}
+doctran = {version = "^0.0.5", optional = true}
```

Comment:
Just include this in the notebook rather than adding to `pyproject.toml`

```
! pip install doctran
```

Reply:
How does it get loaded in the parser modules if it's not included as a dependency?